forui/lib/src/localizations/localizations.dart

```diff
@@ -0,0 +1,699 @@
+    Locale('as'),
+    Locale('az'),
+    Locale('be'),
+    Locale('bg'),
+    Locale('bn'),
+    Locale('bs'),
+    Locale('ca'),
+    Locale('cs'),
+    Locale('cy'),
+    Locale('da'),
+    Locale('de'),
+    Locale('de', 'CH'),
+    Locale('el'),
+    Locale('en'),
+    Locale('en', 'AU'),
+    Locale('en', 'CA'),
+    Locale('en', 'GB'),
+    Locale('en', 'IE'),
+    Locale('en', 'IN'),
+    Locale('en', 'NZ'),
+    Locale('en', 'SG'),
+    Locale('en', 'ZA'),
+    Locale('es'),
+    Locale('es', '419'),
+    Locale('es', 'AR'),
+    Locale('es', 'BO'),
+    Locale('es', 'CL'),
+    Locale('es', 'CO'),
+    Locale('es', 'CR'),
+    Locale('es', 'DO'),
+    Locale('es', 'EC'),
+    Locale('es', 'GT'),
+    Locale('es', 'HN'),
+    Locale('es', 'MX'),
+    Locale('es', 'NI'),
+    Locale('es', 'PA'),
+    Locale('es', 'PE'),
+    Locale('es', 'PR'),
+    Locale('es', 'PY'),
+    Locale('es', 'SV'),
+    Locale('es', 'US'),
+    Locale('es', 'UY'),
+    Locale('es', 'VE'),
+    Locale('et'),
+    Locale('eu'),
+    Locale('fa'),
+    Locale('fi'),
+    Locale('fil'),
+    Locale('fr'),
+    Locale('fr', 'CA'),
+    Locale('gl'),
+    Locale('gsw'),
+    Locale('gu'),
+    Locale('he'),
+    Locale('hi'),
+    Locale('hr'),
+    Locale('hu'),
+    Locale('hy'),
+    Locale('id'),
+    Locale('is'),
+    Locale('it'),
+    Locale('ja'),
+    Locale('ka'),
+    Locale('kk'),
+    Locale('km'),
+    Locale('kn'),
+    Locale('ko'),
+    Locale('ky'),
+    Locale('lo'),
+    Locale('lt'),
+    Locale('lv'),
+    Locale('mk'),
+    Locale('ml'),
+    Locale('mn'),
+    Locale('mr'),
+    Locale('ms'),
+    Locale('my'),
+    Locale('nb'),
+    Locale('ne'),
+    Locale('nl'),
+    Locale('no'),
+    Locale('or'),
+    Locale('pa'),
+    Locale('pl'),
+    Locale('ps'),
+    Locale('pt'),
+    Locale('pt', 'PT'),
+    Locale('ro'),
+    Locale('ru'),
+    Locale('si'),
+    Locale('sk'),
+    Locale('sl'),
+    Locale('sq'),
+    Locale('sr'),
+    Locale.fromSubtags(languageCode: 'sr', scriptCode: 'Latn'),
+    Locale('sv'),
+    Locale('sw'),
+    Locale('ta'),
+    Locale('te'),
+    Locale('th'),
+    Locale('tl'),
+    Locale('tr'),
+    Locale('uk'),
+    Locale('ur'),
+    Locale('uz'),
+    Locale('vi'),
+    Locale('zh'),
+    Locale('zh', 'HK'),
+    Locale('zh', 'TW'),
+    Locale('zu')
+  ];
+
+  /// The full date.
+  ///
+  /// In en, this message translates to:
+  /// **'{date}'**
+  String fullDate(DateTime date);
+
+  /// The year.
+  ///
+  /// In en, this message translates to:
+  /// **'{date}'**
+  String year(DateTime date);
+
+  /// The year and month.
+  ///
+  /// In en, this message translates to:
+  /// **'{date}'**
+  String yearMonth(DateTime date);
+
+  /// The abbreviated month.
+  ///
+  /// In en, this message translates to:
+  /// **'{date}'**
+  String abbreviatedMonth(DateTime date);
+
+  /// The day of the month.
+  ///
+  /// In en, this message translates to:
+  /// **'{date}'**
+  String day(DateTime date);
```

Comment:
So we're doing this because flutter doesn't have support for this natively? As much as possible, I don't think we should be maintaining any of these. This is quite beyond the scope of Forui imo.

Reply:
The actual implementations are actually generated by flutter, internally backed by the `intl` package. It technically is supported via `MaterialLocalizations`, Material's own localizations, but I don't it's a good idea to depend on that directly. 

Also, this class is generated :P